chapters/ch44.md

```diff
@@ -1,85 +1,71 @@
 
---> _replace THIS LINE by your translation for the above line_
-
+Làm thế nào để bạn tính toán được các "arg max" ở trên? Nếu tiếng Anh có 50.000 từ thì N (em nghĩ đoạn này bị thiếu?)
+có (50.000) ​<sup>N</sup> các câu có thể có độ dài ​*N*—​ quá nhiều để liệt kê ra một cách triệt để.
 
 > So, you need to apply an approximate search algorithm, to try to find the value of ​*S* that
 > optimizes (maximizes) Score​<sub>A​</sub>(​*S*). One example search algorithm is “beam search,” which keeps only ​*K* top candidates during the search process. (For the purposes of this chapter, you don’t need to understand the details of beam search.) Algorithms like this are not guaranteed to find the value of ​*S* ​that maximizes Score​<sub>A​</sub>(​*S*).
 
---> _replace THIS LINE by your translation for the above line_
-
+Vì vậy, bạn cần áp dụng thuật toán tìm kiếm gần đúng, để cố gắng tìm giá trị của ​*S* để tối ưu hóa (tối đa hóa) Điểm​<sub>A​</sub>(​*S*). Ví dụ với thuật toán tìm kiếm chùm tia (beam search), thuật toán này chỉ giữ các ứng viên hàng đầu ​*K* trong quá trình tìm kiếm. (Đối với mục đích của chương này, bạn không cần phải hiểu chi tiết về tìm kiếm chùm tia.) Các thuật toán như thế này không đảm bảo được việc tìm giá trị của ​*S* ​ mà tối đa hóa Điểm​<sub>A​</sub>(​*S*).
 
 > Suppose that an audio clip ​*A​* records someone saying “I love machine learning.” But instead of outputting the correct transcription, your system outputs the incorrect “I love robots.” There are now two possibilities for what went wrong:
 
---> _replace THIS LINE by your translation for the above line_
-
+Giả sử rằng một đoạn âm thanh ​*A​* ghi lại một người nào đó nói rằng "Tôi yêu thích học máy". Tuy nhiên, thay vì xuất ra chính xác, hệ thống của bạn lại đưa ra một phiên bản không chính xác "Tôi yêu thích người máy". Có hai khả năng giải thích cho việc sai này:
 
 > 1. **Search algorithm problem​**. The approximate search algorithm (beam search) failed to find the value of ​*S​* that maximizes Score​<sub>A​</sub>(​*S*).
 
---> _replace THIS LINE by your translation for the above line_
-
+1. **Vấn đề về thuật toán tìm kiếm​**. Thuật toán tìm kiếm gần đúng (tìm kiếm chùm tia) không thể tìm thấy giá trị của ​*S​* thỏa mãn việc tối đa hóa Điểm​<sub>A​</sub>(​*S*).
 
 > 2. **Objective (scoring function) problem.**​ Our estimates for Score​<sub>A​</sub>(​*S*) = P(​*S​*|​*A​*) were inaccurate. In particular, our choice of Score​<sub>A​</sub>(​*S*) failed to recognize that “I love machine learning” is the correct transcription.
 
---> _replace THIS LINE by your translation for the above line_
-
+2. **Vấn đề hàm cho điểm mục tiêu.** [cái này em chưa tìm được từ ngữ thích hợp để diễn tả] Ước tính của chúng tôi về Điểm​<sub>A​</sub>(​*S*) = P(​*S​*|​*A​*) không chính xác. Cụ thể, Điểm​<sub>A​</sub>(​*S*) chúng tôi chọn đã không thể nhận ra "Tôi yêu thích học máy" là chính xác.
```

Comment:
Từ "problem" ở đây có nghĩa là "bài toán", "objective problem" là "bài toán mục tiêu", "scoring function" em cũng để trong ngoặc đơn và dịch là "hàm tính điểm". Cả đoạn sẽ là:
> ** Bài toán mục tiêu (hàm tính điểm)**
* "estimaste" = "ước lượng"
* "we" trong văn viết có nghĩa là "chúng ta", ý bao gồm và người đọc và người viết.

> Cụ thể, _cách tính_ Điểm​<sub>A​</sub>(​*S*) ~~của ch chọn đã không thể~~ _của chúng ta thất bại trong việc xác định_ ~~nhận ra~~ "Tôi yêu thích học máy" là _cách phiên âm_ chính xác.